Tasks/AzureKeyVaultV1/operations/KeyVault.ts

```diff
@@ -192,28 +192,35 @@ export class KeyVault {
             return;
         }
 
+        // Support multiple stages using different key vaults with the same secret name but with different version identifiers
+        let secretNameWithoutVersion = secretName.split("/")[0];
```

Comment:
Does this call this.keyVaultClient.getSecretValue work fine for versioned secrets?

Reply:
@bansalaseem You mean for one that does not have a '/' in them - Yes. The test does cover for both cases 
`tr.setInput("SecretsFilter", "secret1, secret2, secret3/versionIdentifierGuid ")`

Reply:
@bansalaseem Yes that does work - It does even today if you were to add a version identifier to the end.  The [REST API format](https://docs.microsoft.com/en-us/rest/api/keyvault/getsecret/getsecret) is such which makes it work

GET {vaultBaseUrl}/secrets/{secret-name}/{secret-version}